src/Microsoft.ML.CpuMath/AlignedArray.cs

```diff
@@ -110,15 +108,21 @@ private void MoveData(int newBase)
             }
         }
 
-        public void CopyTo(Span<Float> dst, int index, int count)
+        public void CopyTo(Span<float> dst)
+        {
+            Contracts.Assert(dst != null);
+            Items.AsSpan().CopyTo(dst);
```

Comment:
This doesn't look correct. Aligned array of size 100 could contain 101 elements. Please use my code to do the copy.